source/camel-case.d.ts

```diff
@@ -0,0 +1,49 @@
+import {WordSeparators} from './delimiter-case';
```

Comment:
`WordSeparators` should be in `utilities.d.ts`. Types should ideally not import from each other.